### DISCUSSION TITLE
ground/terrain height?

### DISCUSSION THREAD

**Discussion by beanhedge** [axaq/traviso.js#23] [Q&A] [answered]
Hi, I see on the Readme that these two features are in development:

> Built-in support for block-like tiles
> Ground/terrain height

The game I'm making depends on these features.  Is this still coming in the future?  If not, do you have any suggestions on how they should be implemented?  Thanks!

**Comment by axaq** [comment, marked as answer]
Hi @beanhedge , yes you are right, those features are definitely in my plans of extending traviso, but my daily job doesn't leave me much time to focus on this at the moment. If you want to go ahead and try it yourself, I would recommend two ways.

* you can either implement layers of maps (floor layer, elevation layer, objects layer)
* or you can treat the elevation as part of the objects layer. You would need to introduce a new type of object called something like terrain so that you can make your characters interact with them different. 

hope this helps.
